turtle_graphics/index.js

```diff
@@ -0,0 +1,170 @@
+class Turtle{
+    constructor(x,y){
+        this.x = x;
+        this.y = y;
+        this.direction = "EAST"
+        this.totalPoint = [[x,y]];
+        this.xstart = x;
+        this.ystart = y;
+    }
+    forward(num){
+        switch(this.direction){
+            case "EAST":
```

Comment:
repeating values should be declared as constants, or class variables, instead of creating new strings